src/plugins/data/common/index_patterns/index_patterns/index_patterns.ts

```diff
@@ -276,11 +284,20 @@ export class IndexPatternsService {
     title: string,
     options: GetFieldsOptions
   ) => {
+    // eslint-disable-next-line no-console
+    console.log('refreshFieldSpecMap start');
     const scriptdFields = Object.values(fields).filter((field) => field.scripted);
     try {
       const newFields = await this.getFieldsForWildcard(options);
```

Comment:
@mattkime here it is where it fails if it's flaky

https://kibana-ci.elastic.co/job/kibana+flaky-test-suite-runner/966/testReport/junit/Chrome%20UI%20Functional%20Tests/test_functional_apps_discover__field_data·js/Kibana_Pipeline___agent_2___discover_app_discover_tab_field_data_search_php_should_show_the_correct_hit_count_7/

Reply:
@kertal I'm not seeing any logs at that URL, perhaps I'm missing something

Reply:
true, can you open 
https://kibana-ci.elastic.co/job/kibana+flaky-test-suite-runner/966/testReport/
and select one of the "discover app discover tab field data search php should show the correct hit count" errors?